api/types/types.go

```diff
@@ -115,6 +115,21 @@ type Version struct {
 	BuildTime     string `json:",omitempty"`
 }
 
+// ClientVersion contains response of Client API:
+type ClientVersion struct {
```

Comment:
Same for this `ClientVersion` type; it's only used in the CLI (not in the API / API-client); I think we should move this.